webrtc/interfaces.html

```diff
@@ -19,7 +19,8 @@
           modulusLength: 2048,
           publicExponent: new Uint8Array([1, 0, 1]),
           hash: 'SHA-256'
-        });
+        })
+        .catch(() => {}); // ignore error
```

Comment:
Note that this will cause doIdlTests to later be called with undefined, while the early return will result in null. That's OK, both are falsy.